numba/targets/arrayobj.py

```diff
@@ -1975,6 +1975,10 @@ def dtype_type(context, builder, dtypety, dtypeval):
     # Just return a dummy opaque value
     return context.get_dummy_value()
 
+@lower_getattr(types.DType, 'kind')
```

Comment:
Is this used?

Reply:
This was used in a previous implementation to find out if a type is float or integer (and get its min/max). I left this here since it doesn't hurt :)